docs/admins/howto/new-hub.rst

```diff
@@ -54,25 +64,40 @@ we provide that sets up a blank hub that can be customized.
    #. Click on the three horizontal bar icon at the top left corner.
    #. Access "Compute Engine" -> "VM instances" -> and search for "nfs-server-01". 
    #. Select "Open in browser window" option to access NFS server via GUI.
-	
-   Create ``staging`` and ``prod``  directories owned by ``1000:1000`` under
-   ``/export/homedirs-other-2020-07-29/<hubname>``. The path *might* differ if
-   your hub has special home directory storage needs. Consult admins if that's
-   the case. Here is the command to create the directory with appropriate permissions,
+   #. Back in the NFS server shell, mount the new share:
 
+   .. code:: bash
+
+      mkdir /export/<hubname>-filestore
+      mount <filestore share IP>/shares /export/<hubname>-filestore
+
+   #. Create ``staging`` and ``prod``  directories owned by ``1000:1000`` under
+   ``/export/<hubname>-filestore/<hubname>``. The path *might* differ if
+   your hub has special home directory storage needs. Consult admins if that's
+   the case. Here is the command to create the directory with appropriate permissions:
+   
    .. code:: bash
 
       install -d -o 1000 -g 1000 \
-        /export/homedirs-other-2020-07-29/<hubname>/staging \
-        /export/homedirs-other-2020-07-29/<hubname>/prod
+        /export/<hubname>-filestore/<hubname>/staging \
+        /export/<hubname>-filestore/<hubname>/prod
 		
-     Check whether the directories have permissions similar to the below directories,
-	
-      .. code:: bash
-		 
-         drwxr-xr-x 4 ubuntu ubuntu     45 Nov  3 20:33 a11y
-	 drwxr-xr-x 4 ubuntu ubuntu     33 Jan  4  2022 astro
-	 drwxr-xr-x 4 ubuntu ubuntu  16384 Aug 16 18:45 biology
+   #. Check whether the directories have permissions similar to the below directories:
+
+   .. code:: bash
+
+         drwxr-xr-x 4 ubuntu ubuntu     45 Nov  3 20:33 a11y-filestore
+	 drwxr-xr-x 4 ubuntu ubuntu     33 Jan  4  2022 astro-filestore
+	 drwxr-xr-x 4 ubuntu ubuntu  16384 Aug 16 18:45 biology-filestore
+
+   #. Now we need to set ``ROOT_SQUASH`` on the newly created mount. In the 
+      ``datahub/deployments/<hubname>/config/filestore/`` directory is a file named 
+      ``<hub-name>-squash-flags.json``. You will use this file when running the following ``gcloud`` 
```

Comment:
With the latest changes you can remove `<hub-name>-` from `<hub-name>-squash-flags.json`.